Library/Homebrew/sbom.rb

```diff
@@ -10,9 +10,7 @@
 # Rather than calling `new` directly, use one of the class methods like {SBOM.create}.
 class SBOM
   FILENAME = "sbom.spdx.json"
-  SCHEMA_URL = "https://spdx.github.io/spdx-3-model/model.jsonld"
-  SCHEMA_FILENAME = "sbom.spdx.schema.3.json"
-  SCHEMA_CACHE_TARGET = (HOMEBREW_CACHE/"sbom/#{SCHEMA_FILENAME}").freeze
+  SCHEMA_FILE = (HOMEBREW_LIBRARY_PATH/"data/schemas/sbom.spdx.schema.3.json").freeze
```

Comment:
Do we expect to have more schemas coming here?

Reply:
I think it would be a good idea to add the OCI ones here.

Reply:
@Bo98 Cool, thanks.